interfaces/api/dependencies.py

```
@@ -310,22 +320,112 @@ def get_hosted_write_service() -> HostedWriteService:
     )
 
 
+class MultiModelRouter(LLMService):
+    """一个能够同时持有多个 Provider 实例并根据模型配置进行路由的门面（Facade）。
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)